app/code/Magento/CheckoutAgreements/etc/adminhtml/system.xml

```diff
@@ -13,6 +13,10 @@
                     <label>Enable Terms and Conditions</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="enable_editor" translate="label" type="select" sortOrder="25" showInDefault="1" showInWebsite="0" showInStore="0">
```

Comment:
please add the introduced configuration node to config.xml with the default value of 0.
I would also suggest renaming the setting to `enable_wysiwyg_editor` and label to "Enable WYSIWYG Editor".